scripts/helper-image.sh

```diff
@@ -3,11 +3,14 @@
 set -eou pipefail
 
 help() {
-  echo "Usage: $0 --hostname [hostname] --identity [1|2|3]"
+  echo "Usage: $0 --hostname [hostname] --identity [1|2|3] --platform [platform]"
   echo "- hostname: public hostname that will appear on TLS certificate"
   echo "- identity: helper identity to be fixed inside the IPA build"
+  echo "- platform: the platform for the helper image, defaults to linux/amd64"
 }
 
+platform="linux/amd64"
```

Comment:
that was a gotcha for me - docker uses supervisor's platform when building images, so I couldn't run them on Linux later. 

Avoid generating those images on MacOS - it is extremely slow (but working). Better to run this script on a Linux machine